### DISCUSSION TITLE
Running under Elementary v8.0 (Ubuntu 24.02 LTS), or not actually

### DISCUSSION THREAD

**Discussion by doobes** [TriliumNext/Trilium#6040] [Ideas] [answered]
So,

I've just installed EoS v8.0 on an old Dell XPS 13 laptop. I've had EoS installed on this machine for some time now.

Unfortunately Trilium refuses to run due to the recenting invoked Ubuntu sandboxing scheme.

Fuse 2 is installed on the machine and make no difference.

This is what I'm getting:

```
chris@XPS13:~/Applications/trilium$ ./trilium
[6109:1101/092757.811643:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/chris/Applications/trilium/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap (core dumped)
chris@XPS13:~/Applications/trilium$ 
```

Spelunking around, I can turn off the sandboxing by running these two commands:

sudo sysctl -w kernel.apparmor_restrict_unprivileged_unconfined=0
sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0

Trouble is, this must be done every time I reboot.

The dev's at EoS indicate this is an app issue.  I do know that the Ondsel AppImage will run without the bypass.

Any thoughts about a permanent solution would be greatly appreciated.

cheers

chris

**Comment by eliandoran** [comment]
@doobes , is Elementary OS 8 a beta build? If so, please provide a link to an ISO so that I can test if the issue is reproducible on a virtual machine. 

Also, which installation method was used for the application? .deb or .zip?

**Comment by doobes** [comment]
Elementary OS v8.0 officially released, but unfortunately only for sponsors.

I don't think they want me posting the sponsored links.

As soon as there is an official public release, I'll post it. 

They have declared this to be an "upstream" issue with Ubuntu 24.02 LTS.

I'm using v.90.4 at present.  I'll try the latest release tomorrow.

cheers,

**Reply by eliandoran** [reply]
Seems to be an issue with all the Electron-based applications on newer Ubuntu versions, I will have a look.

**Reply by doobes** [reply]
Thank you

**Comment by doobes** [comment, marked as answer]
I'm in the process of standing up an Ubuntu 24.02 LTS VM to test.

In any event, an answer is here: [https://askubuntu.com/questions/1511654/how-to-make-balena-etcher-and-other-appimages-work-in-ubuntu-24-04](url)
